tex/generic/pgf/math/pgfmathfloat.code.tex

```diff
@@ -1312,7 +1312,7 @@
 % The numerator and denominator is always a number (not empty)
 \def\pgfmathfloatgetfrac#1{%
     \pgfutil@ifundefined{pgfmathfloatmultiply@}{%
-        \pgfmath@PackageError{Sorry, the number format 'frac' requires '\string\usetikzlibrary{fpu}' (and, optionally, \string\usepackage{fp}) in order to work correctly}%
+        \pgfmath@error{Sorry, the number format 'frac' requires '\string\usetikzlibrary{fpu}' (and, optionally, \string\usepackage{fp}) in order to work correctly}%
```

Comment:
```suggestion
        \pgfmath@error{Sorry, the number format 'frac' requires '\string\usetikzlibrary{fpu}' (and, optionally, \string\usepackage{fp}) in order to work correctly}{}%
```

`\pgfmath@error` need two args. I find more wrong one-arg usages of `\pgfmath@error` in code base ...

Reply:
Guess if I copied the example from the wrong code ;-)

Fixed (elsewhere too)

Reply:
Many thanks. Would you be kind enough to fix the last three remaining occurrences in [`pgflibraryluamath.code.tex`](https://github.com/pgf-tikz/pgf/blob/master/tex/generic/pgf/libraries/luamath/pgflibraryluamath.code.tex)?

Reply:
Oops, grepped one directory too deep.  Done